ui/main/src/app/services/userPermissions/UserPermissionsService.ts

```
@@ -32,7 +31,7 @@ export class UserPermissionsService {
     public static doesTheUserHavePermissionToDeleteCard(user: UserWithPerimeters, card: Card): boolean {
         let permission = false;
         if (
-            !UsersService.hasCurrentUserAnyPermission([PermissionEnum.READONLY]) &&
+            !this.isUserReadonly(user) &&
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)